Jenkinsfile

```diff
@@ -114,15 +114,15 @@ if ( config.branches.containsKey(env.BRANCH_NAME) ) {
             node {
                 unstash 'scripts'
                 try {
-                    utils.pushPrivateReg(region.registry_port)
+                    utils.pushPrivateReg(region.registry_port, branchConfig.apps)
                 } catch(err) {
                     utils.ircNotification(config, [stage: stageName, status: 'failure'])
                     throw err
                 }
             }
         }
         for (appname in branchConfig.apps) {
-            def appURL = "https://${appname}.${region.name}.moz.works"
+            def appURL = "https://${appname}${branchConfig.app_name_suffix}.${region.name}.moz.works"
```

Comment:
The "-deis" `app_name_suffix` isn't globally required, and in fact in the two places it is required it's due to an artifact of a past misunderstanding and should probably be fixed. There are some nuances there that are probably best discussed over a higher bandwidth channel though.